### ISSUE TITLE
order by collection count

### ISSUE THREAD

**Issue by mathroc** [graphile-contrib/pg-order-by-related#6]
example:

```gql
{
  allPost(orderBy: COMMENTS_COUNT_DESC) { id name { comments { totalCount } } }
}
```

would be nice if the generated query don't join twice on comments to get the totalCount and sort by it

**Comment by mattbretl** [comment]
Where is `COMMENTS_COUNT_DESC` coming from? Does that exist in your schema, or is that a feature request (aggregation functions on relations)?

**Comment by mathroc** [comment]
that's a feature request yes :) sorry it wasn't clear

**Comment by mattbretl** [comment]
I just published `1.0.0-beta.2` which includes this.

**Comment by mattbretl** [comment]
> would be nice if the generated query don't join twice on comments to get the totalCount and sort by it

There's really only one way to implement this given the current plugin architecture. If that changes, we can revisit.
